### PR TITLE
updated add player UI

### LINKED ISSUE #36: UI for Add player

### PR DESCRIPTION
**Pull request by yonbergman**
Fixed #36 
tekken3

### CODE REVIEW

**Comment by udiWertheimer** [comment]
ole

**Comment by yonbergman** [comment]
Added filtering

**Comment by udiWertheimer** [comment]
ole again

File: app/src/main/java/com/iic/mokojin/AddPlayerActivity.java [hunk 1]
```diff
@@ -55,36 +66,21 @@ protected void onCreate(Bundle savedInstanceState) {
...
     public static class AddPlayerFragment extends Fragment {
 
-        private ParseQueryAdapter<Person> mAdapter;
+        private PersonQueryAdapter mAdapter;
+        private ProgressHudDialog mProgressDialog;
 
         public AddPlayerFragment() {
-            setHasOptionsMenu(true);
-        }
+                    }
```
Comment (giladgo) [inline app/src/main/java/com/iic/mokojin/AddPlayerActivity.java:73]: indentation?

**Comment by giladgo** [comment]
:football:

**Pull request merged**